ui/editor/src/view.ts

```diff
@@ -222,6 +223,10 @@ function controls(ctrl: EditorCtrl, state: EditorState): VNode {
                 attrs: { id: 'variants' },
                 on: {
                   change(e) {
+                    const option = (e.target as HTMLSelectElement).item(
+                      (e.target as HTMLSelectElement).selectedIndex,
+                    );
+                    if (option) ctrl.setVariant(option.text.slice(10));
```

Comment:
this is verbose and fragile. Instead we could have a `['chess960', 'Chess 960']` option and set the ctrl.rule and ctrl.isChess960 flags accordingly